client/app/visualizations/sankey/d3sankey.js

```diff
@@ -166,39 +159,6 @@ function Sankey() {
       });
     }
 
-    function resolveCollisions() {
-      nodesByBreadth.forEach(nodes => {
```

Comment:
Duplicate code fragment (IntelliJ IDEA became smarter after update and managed to find it this time :grin:)